python/graphscope/nx/tests/test_copy_on_write.py

```diff
@@ -353,7 +362,7 @@ def test_average_shortest_path_length(self):
 
     def test_bfs_edges(self):
         ret = nx.builtin.bfs_edges(self.SG, 1, depth_limit=10)
-        assert sorted(ret) == [[1, 3], [3, 5], [4, 6], [5, 4]]
+        assert sorted(ret) == [[1, 2], [1, 3], [3, 5], [5, 4], [5, 6]]
```

Comment:
ditto

Reply:
There was a bug in bfs_edges and fixed, 1[[1, 2], [1, 3], [3, 5], [5, 4], [5, 6]]` is the correct answer.